met-web/src/components/engagement/admin/create/authoring/AuthoringTemplate.tsx

```diff
@@ -0,0 +1,192 @@
+    const [singleContentValues, setSingleContentValues] = useState({ ...defaultTabValues, heading: '' });
+    const tenant = useAppSelector((state) => state.tenant);
+    const languages = useMemo(() => getTenantLanguages(tenant.id), [tenant.id]); // todo: Using tenant language list until language data is integrated with the engagement.
+    const authoringRoutes = getAuthoringRoutes(Number(engagementId), tenant);
+    const location = window.location.href;
+    const locationArray = location.split('/');
+    const slug = locationArray[locationArray.length - 1];
+    const pageTitle = authoringRoutes.find((route) => {
+        const pathArray = route.path.split('/');
+        const pathSlug = pathArray[pathArray.length - 1];
+        return pathSlug === slug;
+    })?.name;
+    const {
+        handleSubmit,
+        setValue,
+        watch,
+        control,
+        formState: { isDirty, isValid, isSubmitting },
+    } = useFormContext<EngagementUpdateData>();
+
+    // Set hidden values for form data
+    setValue('id', Number(engagementId));
+
+    const eyebrowTextStyles = {
+        fontSize: '0.9rem',
+        marginBottom: '1rem',
+    };
+
+    // If switching from single to multiple, add "tab 2". If switching from multiple to single, remove all values but index 0.
+    const handleTabsRadio = (event: React.ChangeEvent<HTMLInputElement>) => {
+        const newTabsRadioValue = event.target.value;
+        if ('true' === newTabsRadioValue) {
+            setTabs([
+                { ...tabs[0], heading: singleContentValues.heading ? singleContentValues.heading : 'Tab 1' },
+                { ...defaultTabValues, heading: 'Tab 2' },
+            ]);
+            setContentTabsEnabled('true');
+        } else {
+            setSingleContentValues({ ...tabs[0], heading: 'Tab 1' !== tabs[0].heading ? tabs[0].heading : '' });
+            setTabs([tabs[0]]);
+            setContentTabsEnabled('false');
+        }
+    };
+
+    return (
+        <ResponsiveContainer>
+            <AutoBreadcrumbs />
+            <Suspense>
+                <Await resolve={engagement}>
+                    {(engagement: Engagement) => (
+                        <div style={{ marginTop: '2rem' }}>
+                            <StatusLabel text={EngagementStatus[engagement?.status_id]} completed={false} />
+                            {/* todo: For the section status label when it's ready */}
+                            {/* <StatusLabel text={'Insert Section Status Text Here'} completed={'Insert Completed Boolean Here'} /> */}
+                        </div>
+                    )}
+                </Await>
+            </Suspense>
+            <h1 style={{ marginTop: '0.5rem', paddingBottom: '1rem' }}>{pageTitle}</h1>
+            <SystemMessage status="warning">
+                Under construction - the settings in this section have no effect.
+            </SystemMessage>
+            {'details' === slug && (
```

Comment:
Just curious why this markup lives here instead of in AuthoringDetails.jsx. Was it easier to style or something?

Reply:
It's just about page positioning. For the details page specifically, there is a "content configuration" that appears above the current language h2. We could change that and have the content local to the Details component.

Reply:
Just looked back at the designs and that makes sense!  Thanks for explaining